caddytest/integration/caddyfile_adapt/global_options.txt

```diff
@@ -0,0 +1,71 @@
+{
+	debug
+	http_port 8080
+	https_port 8443
+	default_sni localhost
+	order root first
+	storage file_system {
+		root /data
+	}
+	acme_ca https://example.com
+	acme_ca_root /path/to/ca.crt
+	email test@example.com
+	admin off
+	on_demand_tls {
+		ask https://example.com
+		interval 30s
+		burst 20
+	}
+	local_certs
+	key_type ed25519
+}
+
+:80
```

Comment:
Is this intentional?

Reply:
Yeah the first bit is a bunch of global options, then `:80` is the site label.